README.md

```diff
@@ -94,9 +97,16 @@ The list of currently supported accelerators (termed [Execution Providers](./doc
 
 ## Builds and Packages
 
-Official builds are published for the default CPU Provider (Eigen + MLAS), as well as GPU with CUDA. Python packages can be found on PyPi, and C#/C/C++ packages on Nuget. Please view the table on [aka.ms/onnxruntime](https://aka.ms/onnxruntime) for instructions for different build combinations. 
+Official builds are available for:
+* Default CPU Provider (Eigen + MLAS)
+* GPU Provider - NVIDIA CUDA
+  * *note: If your deployment target is Windows, the [DirectML execution provider](./docs/execution_providers/DirectML-ExecutionProvider.md) is recommended for optimal performance and compatibility with a broad set of GPUs. This will be an official package soon. In the meantime, see the build instructions at [BUILD.md](./BUILD.md#directml).* 
+
+Python packages can be found on PyPi, and C#/C/C++ packages on Nuget. Please view the table on [aka.ms/onnxruntime](https://aka.ms/onnxruntime) for instructions for different build combinations. 
 
-For additional build flavors and/or dockerfiles, please see [BUILD.md](BUILD.md). For production scenarios, it's strongly recommended to build only from an [official release branch](https://github.com/microsoft/onnxruntime/releases).
+For additional build flavors and/or dockerfiles, please carefully read through [BUILD.md](./BUILD.md). If you encounter problems, please provide as much information as possible when filing an Issue. 
```

Comment:
Issue -> issue. Also, let's link to https://github.com/Microsoft/onnxruntime/issues to make it clear where to file issues.